packages/cozy-harvest-lib/src/locales/en.json

```
@@ -77,20 +77,20 @@
         "description": "It seems that this account's destination folder has been deleted. Please restore it by disconnecting this account and then reconnect again."
       },
       "TERMS_VERSION_MISMATCH": {
-        "title": "Unexpected Terms Of Service version",
+        "title": "Latest Terms of Service non accepted",
         "description": "%{name} seems to have updated its Terms Of Service. Please check that the service is up to date. It this error still occurs, please contact us at <a href=\"mailto:contact@cozycloud.cc\">contact@cozycloud.cc</a>."
       },
       "UNKNOWN_ERROR": {
         "title": "Connection error",
         "description": "An unknown error has occurred."
       },
       "USER_ACTION_NEEDED": {
-        "title": "Action needed on the provider",
-        "description": "It seems that the [%{name}](%{link}) website requires you to log in and to do a specific action. Please rerun the connector once you have settled the issue on the website."
+        "title": "Action needed on provider website",
```

```suggestion
        "title": "Action needed on the provider's website",
```

I am not sure but it sounds less robotic

---

sounds better yes